bindings/python/iota_sdk/client/responses.py

```diff
@@ -0,0 +1,359 @@
+
+    Attributes:
+        slot: The slot index for which the congestion estimate is provided.
+        ready: Indicates if a node is ready to issue a block in a current congestion or should wait.
+        reference_mana_cost: The cost in mana for issuing a block in a current congestion estimated based on RMC and slot index.
+        block_issuance_credits: The Block Issuance Credits of the requested account.
+    """
+    slot: SlotIndex
+    ready: bool
+    reference_mana_cost: int = field(metadata=config(
+        encoder=str
+    ))
+    block_issuance_credits: int = field(metadata=config(
+        encoder=str
+    ))
+
+
+@json
+@dataclass
+class ManaRewardsResponse:
+    """The mana rewards of an account or delegation output.
+    Response of GET /api/core/v3/rewards/{outputId}.
+
+    Attributes:
+        start_epoch: First epoch for which rewards can be claimed. This value is useful for checking if rewards have expired (by comparing against the staking or delegation start) or would expire soon (by checking its relation to the rewards retention period).
+        end_epoch: Last epoch for which rewards can be claimed.
+        rewards: Amount of totally available decayed rewards the requested output may claim.
+        latest_committed_epoch_pool_rewards: Rewards of the latest committed epoch of the staking pool to which this validator or delegator belongs. The ratio of this value and the maximally possible rewards for the latest committed epoch can be used to determine how well the validator of this staking pool performed in that epoch. Note that if the pool was not part of the committee in the latest committed epoch, this value is 0.
+    """
+    start_epoch: EpochIndex
+    end_epoch: EpochIndex
+    rewards: int = field(metadata=config(
+        encoder=str
+    ))
+    latest_committed_epoch_pool_rewards: int = field(metadata=config(
+        encoder=str
+    ))
+
+
+@json
+@dataclass
+class ProtocolParametersResponse:
+    """Protocol Parameters with start epoch.
+
+    Attributes:
+        start_epoch: The start epoch of the set of protocol parameters.
+        parameters: The protocol parameters.
+    """
+    start_epoch: EpochIndex
+    parameters: ProtocolParameters
+
+
+# TODO: rename sdk-wide to `NodeInfoResponse`?
+@json
+@dataclass
+class InfoResponse:
+    """General information about the node.
+    GET /api/core/v3/info.
+
+    Attributes:
+        name: The name of the node (e.g. Hornet).
+        version: The semantic version of the node.
+        status: The status of the node.
+        metrics: Node metrics.
+        protocol_parameters: Supported protocol versions by the node.
+        base_token: Gives info about the base token the network uses.
+    """
+    name: str
+    version: str
+    status: NodeInfoStatus
+    metrics: NodeInfoMetrics
+    protocol_parameters: List[ProtocolParametersResponse]
+    base_token: NodeInfoBaseToken
+
+
+# TODO: rename sdk-wide to `NodeInfoResponseWithUrl`?
```

Comment:
reminder to create an issue

Reply:
https://github.com/iotaledger/iota-sdk/issues/1977